packages/venia-concept/src/components/Gallery/item.js

```diff
@@ -39,7 +39,9 @@ class GalleryItem extends Component {
         item: shape({
             id: number.isRequired,
             name: string.isRequired,
-            small_image: string.isRequired,
+            small_image: shape({
+                url: string.isRequired
+            }),
```

Comment:
```suggestion
            }).isRequired,
```

Because you're accessing `small_image.url`, `renderImage()` will throw if it's not an object.

Reply:
Added